distmv/normal.go

```diff
@@ -77,6 +77,33 @@ func NewNormalChol(mu []float64, chol *mat64.Cholesky, src *rand.Rand) *Normal {
 	return n
 }
 
+// NewNormalPrecision creates a new Normal distribution with the given mean and
+// precision matrix (inverse of the covariance matrix). NewNormalPrecision
+// panics if len(mu) is not equal to prec.Symmetric(). If the precision matrix
+// is not positive-definite, NewNormalPrecision returns {nil,false}.
```

Comment:
Rephrase the return to take into account return parameter names and avoid braces.